mysql/datadog_checks/mysql/config_models/instance.py

```diff
@@ -98,6 +98,7 @@ class Config:
 
     ca: Optional[str]
     cert: Optional[str]
+    check_hostname: Optional[bool]
```

Comment:
Where is this actually used in the code?

Reply:
Code wise I am adding a variable in the conf yaml file while also keeping it as an optional setting with default of true to allow DBM on MySQL to happen. In some cases, due to which db type they are using for MySQL, the permissions for the connection can strict as I believe the hostname must be the same as the one set in certificate but setting this to false can prevent this checking from happening and allow DBM to monitor their db. But user do have the option to change this value in the conf yaml file. 

The check_hostname is used in the self._config.ssl and then assigned to ssl as a dictionary